backend/services/agentops.py

```
@@ -0,0 +1,748 @@
+        logger.debug(f"agentops_trace_context.get(): {agentops_trace_context.get()}")
+        logger.debug(f"_current_trace_context: {_current_trace_context}")
+        yield None
+        return
+    
+    logger.debug(f"Creating LLM span for model {model} with trace context {trace_context}")
+    
+    # Extract system prompt if present
+    system_prompt = None
+    for msg in messages:
+        if msg.get("role") == "system":
+            system_prompt = msg.get("content", "")
```

Flake8 (7.2.0)

[error] local variable 'system_prompt' is assigned to but never used

(F841)